src/irr/video/COpenGLRenderpassIndependentPipeline.h

```diff
@@ -174,31 +174,6 @@ class COpenGLRenderpassIndependentPipeline : public IGPURenderpassIndependentPip
     const SVAOHash& getVAOHash() const { return m_vaoHashval; }
 
 private:
-    // TODO move GL object creation to driver
-    GLuint createGLobject(uint32_t _ctxID)
```

Comment:
wait, what's going on?

Reply:
nothing, just deleted code not used any more